src/modifier.ts

```diff
@@ -129,3 +129,30 @@ export type Modifiers = Partial<
       | StringModifier]: ModifierValue<T>;
   }
 >;
+
+function pairwiseMerge(modifiers1: Modifiers, modifiers2: Modifiers) {
```

Comment:
Sorry, try to document this also since it's quite a big fn!

Reply:
aye aye!